--- a/usb/55-Argyll.rules
+++ b/usb/55-Argyll.rules
@@ -130,7 +130,7 @@
 # Set ID_VENDOR and ID_MODEL acording to VID and PID
 #TEST=="/lib/udev/usb-db", IMPORT{program}="usb-db %po"
 ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{program}="usb_id --export %p"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{program}="usb-db %p"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{builtin}="hwdb --susbsystem=usb"

 # Is a color calibration device. 70-uaccess.rules may use this to set TAG+="uaccess",
 # but there is no way to know if this is the case from here.